sceptre/diffing/stack_differ.py

```diff
@@ -9,9 +9,7 @@
 from cfn_tools import ODict
```

Comment:
All of the code removed in this file is because the StackDiffer implemented a more primitive form of the placeholder system introduced in this PR. Now that placeholders can be enabled globally, there's no longer a need to explicitly handle those situations in the StackDiffer.